core/test/truffle-fixture.js

```diff
@@ -0,0 +1,36 @@
+const deployFinder = require("../migrations/2_deploy_finder.js");
+const deployTimer = require("../migrations/3_deploy_timer.js");
+const deployVotingToken = require("../migrations/4_deploy_voting_token.js");
+const deployVoting = require("../migrations/5_deploy_voting.js");
+const deployRegistry = require("../migrations/6_deploy_registry.js");
+const deployFinancialContractsAdmin = require("../migrations/7_deploy_financial_contracts_admin.js");
+const deployStore = require("../migrations/8_deploy_store.js");
+const deployGovernor = require("../migrations/9_deploy_governor.js");
+const deployDesignatedVotingFactory = require("../migrations/10_deploy_designated_voting_factory.js");
+const deploySupportIdentifiers = require("../migrations/11_support_identifiers.js");
+const deployTestnetToken = require("../migrations/12_deploy_testnet_token.js");
+const deployTokenfactory = require("../migrations/13_deploy_tokenfactory.js");
+const deployExpiringMultiPartyCreator = require("../migrations/14_deploy_expiring_multi_party_creator.js");
+const deployLocalWeth = require("../migrations/15_deploy_local_weth.js");
+
+const { getKeysForNetwork } = require("../../common/MigrationUtils.js");
+
+module.exports = async ({ network, web3 }) => {
+  const accounts = await web3.eth.getAccounts();
+  const { deployer } = getKeysForNetwork(network, accounts);
```

Comment:
Note: `keys.deployer` is an address, the deployer that is provided to the migrations is the truffle deployer object. If you don't use or have access to the truffle deployer, then you should probably pass `null` or `undefined` to these functions to signal that this argument isn't being used.